tools/monitor.js

```diff
@@ -0,0 +1,87 @@
+import Arweave from 'arweave';
+import { connect, createDataItemSigner } from '@permaweb/aoconnect';
+import {
+  ANTRegistry,
+  AOProcess,
+  ANT_REGISTRY_ID,
+  ARIO,
+  ARIO_MAINNET_PROCESS_ID,
+} from '@ar.io/sdk';
+import { pLimit } from 'plimit-lit';
+
+const arweave = Arweave.init({
+  host: 'arweave.net',
+  port: 443,
+  protocol: 'https',
+});
+
+async function main() {
+  const jwk = await arweave.wallets.generate();
```

Comment:
provide the wallet via env vars to avoid the compute time for generating wallets

Reply:
[1cc9df8](https://github.com/ar-io/ar-io-ant-registry-process/pull/16/commits/1cc9df8dd28863d53b0fe033e1f34211cabcd15f)